### PR TITLE
rename namespace to isv.

### PR DESCRIPTION
**Pull request by Alice1319**
fixes：
https://github.com/eclipse-xpanse/xpanse/issues/2188

### CODE REVIEW

**Comment by wuzhuanhong** [comment]
LGTM

**Comment by dengyali0125** [comment]
LGTM

**Comment by baixinsui** [comment]
LGTM

**Comment by profoundwu** [comment]
LGTM

File: docs/authentication-authorization.mdx [hunk 1]
```diff
@@ -116,12 +116,12 @@ This option can be found on the UI under - users -> click on user -> Authorizati
...
 
-In order to allow service vendors to manage only their own services, we use ABAC. This is achieved by using `metadata` feature of OIDC to assign `isv` users to a specific namespace.
+In order to allow service vendors to manage only their own services, we use ABAC. This is achieved by using `metadata` feature of OIDC to assign `isv` users to a specific isv value.
 
 ![img.png](../static/img/metadata-isv.png)
 
-While registering a service, the service vendor must specify the same namespace value in the `namespace` field.
-With this, users of another namespace won't be allowed to view, update or delete these service templates.
+While registering a service, the service vendor must specify the same isv value in the `isv` field.
+With this, users of another isv won't be allowed to view, update or delete these service templates.
```
Comment (swaroopar) [inline docs/authentication-authorization.mdx:124]: here the 'isv' must be written as ISV

File: docs/authentication-authorization.mdx [hunk 2]
```diff
@@ -116,12 +116,12 @@ This option can be found on the UI under - users -> click on user -> Authorizati
 
 #### Metadata for ISVs
 
-In order to allow service vendors to manage only their own services, we use ABAC. This is achieved by using `metadata` feature of OIDC to assign `isv` users to a specific namespace.
+In order to allow service vendors to manage only their own services, we use ABAC. This is achieved by using `metadata` feature of OIDC to assign `isv` users to a specific isv value.
```
Comment (swaroopar) [inline docs/authentication-authorization.mdx:119]: users to a specific ISV value.

File: docs/authentication-authorization.mdx [hunk 3]
```diff
@@ -116,12 +116,12 @@ This option can be found on the UI under - users -> click on user -> Authorizati
...
 
 ![img.png](../static/img/metadata-isv.png)
 
-While registering a service, the service vendor must specify the same namespace value in the `namespace` field.
-With this, users of another namespace won't be allowed to view, update or delete these service templates.
+While registering a service, the service vendor must specify the same isv value in the `isv` field.
```
Comment (swaroopar) [inline docs/authentication-authorization.mdx:123]: must specify the same ISV value in the

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: docs/authentication-authorization.mdx (hunk 1 above)
Comment (Alice1319) [reply docs/authentication-authorization.mdx]: updated.

File: docs/authentication-authorization.mdx (hunk 2 above)
Comment (Alice1319) [reply docs/authentication-authorization.mdx]: updated.

File: docs/authentication-authorization.mdx (hunk 3 above)
Comment (Alice1319) [reply docs/authentication-authorization.mdx]: updated.

**Review by swaroopar — APPROVED** [review]

**Pull request merged**